plugins/flutter_aepmessaging/android/build.gradle

```diff
@@ -37,8 +37,6 @@ android {
 
     kotlinOptions {
         jvmTarget = '1.8'
-        languageVersion = "1.4"
```

Comment:
Can you keep this and update it to 1.5 so that it matches what we use in native SDKs?

Reply:
Flutter gives a warning if the Kotlin compiler version is below 2.1.0 which only supports language version of `1.6`. That's the minimum version I can set

Reply:
The flutter bridge library can use features introduced in Kotlin 1.5 (or whatever version is specified). The actual compiler version used for the build is derived from the highest among all modules (the test app in this case). Since Kotlin binaries are backwards compatible (a newer compiler can read older binaries), we should not run into any issues.